snaps_k8s/playbooks/k8/k8_ha_multicluster_loadbalancer_configure.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 ARICENT HOLDINGS LUXEMBOURG SARL and Cable Television
+# Laboratories, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is responsible for deploying Aricent_Iaas environments and
+# Kubernetes Services
+--- 
+- 
+  become: true
+  become_user: root
+  hosts: "{{ host_name }}"
+  vars_files:
+    - "{{ VARIABLE_FILE }}"
```

Comment:
These files no longer exist. You will need to pass in all parameters via the ansible_utils API call

Reply:
Incorporated.